src/cosmoscout/Application.cpp

```diff
@@ -400,6 +397,7 @@ void Application::FrameUpdate() {
 
     // Hide the loading screen after several frames.
     if (GetFrameCount() == mHideLoadingScreenAtFrame) {
+      mGuiManager->getGui()->callJavascript("CosmoScout.initInputs");
```

Comment:
Why do we require this call twice?

Reply:
This was indeed a wrongful double call. Removed the call from `cosmoscout.html`.  
The one from `Application.cpp` should be the 'right' call, as at this point in time all needed html is present in the gui.